src/components/GroupSuggestions/styles.scss

```diff
@@ -0,0 +1,7 @@
+.group-suggestions {
+    background-color: var(--background-gray);
+    border-bottom: 1px solid var(--orange);
+    padding-bottom: 30px;
+    padding-top: 50px;
+    padding-left: 30px;
```

Comment:
If you have multiple paddings in one class you should use `padding: 50px 0 30px 30px`